README.md

```diff
@@ -20,18 +22,28 @@ See details at [wiki](https://github.com/miripiruni/CSScomb/wiki/Requirements).
 
 ```
 cd ~/.vim/bundle
-git clone https://github.com/miripiruni/CSScomb-for-Vim.git
+git clone https://github.com/csscomb/vim-csscomb.git
 ```
 
 ### With Vundle
 Add this to .vimrc:
 ```
-Bundle 'git://github.com/miripiruni/CSScomb-for-Vim.git'
+Bundle 'git://github.com/csscomb/vim-csscomb.git'
+```
+
+### With NeoBundle
+Add this to .vimrc:
+```
+NeoBundle 'csscomb/vim-csscomb', {
+        \ 'build': {
+        \     'unix': 'npm install -g csscomb',
+        \     'mac': 'npm install -g csscomb'
+        \ }}
```

Comment:
I gave it some more thought. `build` is actually run every time plugin is updated. But we don't need to install `csscomb` every time.
Maybe it'll be enough for this plugin to be a wrapper for globally installed package, nothing more? This way user can easily control which version of the package he has installed.
I understand that those lines above are just example, they don't affect the code you wrote, but nonetheless, for those of us who just copy-paste everything we see, should we remove those `build` fields at all?